internal/app/config.go

```diff
@@ -39,6 +40,21 @@ type Auth0Config struct {
 	WebClientID  string
 }
 
+type AuthConfig struct {
+	Domain string `default:"http://localhost:8080"`
+	Key    string
+	Pkix   AuthPkixConfig
```

Comment:
```suggestion
	DN   *AuthDNConfig
```

Make the config optional to use dummy data if there is no DN config